src/screens/SendAmountInput.js

```diff
@@ -138,7 +139,10 @@ class SendAmountInput extends React.Component {
       });
       const { available } = balance;
       const amountAndToken = `${renderValue(available, this.isNFT())} ${this.state.token.symbol}`;
-      return ngettext(msgid`${amountAndToken} available`, `${amountAndToken} available`, available);
+      // Convert BigInt to Number for ngettext - extract as variable for ttag compatibility
+      // This is only used for pluralization so precision loss is acceptable
+      const availableCount = Number(available);
```

Comment:
This may have some issues with big values, maybe we should serialize as string directly?